packages/sommelier.rb

```diff
@@ -44,46 +46,49 @@ class Sommelier < Package
   def self.build
     # There is no good way to checksum the googlesource tgz file, as they appear to be generated on the fly
     # and checksums vary with each download.
-    system 'curl -L https://chromium.googlesource.com/chromiumos/platform2/+archive/c6c3e7aac940d34d3f195bc60d7be2ec19d67d03.tar.gz | tar mzx --warning=no-timestamp'
+    system 'curl -L https://chromium.googlesource.com/chromiumos/platform2/+archive/f3b2e2b6a8327baa2e62ef61036658c258ab4a09.tar.gz | tar mzx --warning=no-timestamp'
     Dir.chdir ("vm_tools/sommelier") do
-
-    # Google's sommelier expects to find virtwl.h in their kernel source includes, but we may not have
-    # set of kernel headers which match, so we just download virtwl.h and then patch the sommelier source
-    # to look for the file locally.
-    ######################### Download virtwl.h from Chromium 5.4 kernel tree ###########################################
-    url_virtwl = "https://chromium.googlesource.com/chromiumos/third_party/kernel/+/5d641a7b7b64664230d2fd2aa1e74dd792b8b7bf/include/uapi/linux/virtwl.h?format=TEXT"
-    uri_virtwl = URI.parse url_virtwl
-    filename_virtwl = 'virtwl.h_base64'
-    sha256sum_virtwl = 'a8215f4946ccf30cbd61fcf2ecc4edfe6d05bffeee0bacadd910455274955446'
-
-    puts "Downloading virtwl".yellow
-    system('curl', '-s', '-C', '-', '--insecure', '-L', '-#', url_virtwl, '-o', filename_virtwl)
-    abort 'Checksum mismatch. :/ Try again.'.lightred unless
-      Digest::SHA256.hexdigest( File.read( filename_virtwl ) ) == sha256sum_virtwl
-    puts "virtwl base64 downloaded".lightgreen
-    FileUtils.mkdir_p 'build/linux'
-    system 'base64 --decode virtwl.h_base64 > build/linux/virtwl.h'
-
+    
+    ## Google's sommelier expects to find virtwl.h in their kernel source includes, but we may not have
+    ## set of kernel headers which match, so we just download virtwl.h and then patch the sommelier source
+    ## to look for the file locally.
+    ########################## Download virtwl.h from Chromium 5.4 kernel tree ###########################################
+    #url_virtwl = "https://chromium.googlesource.com/chromiumos/third_party/kernel/+/5d641a7b7b64664230d2fd2aa1e74dd792b8b7bf/include/uapi/linux/virtwl.h?format=TEXT"
+    #uri_virtwl = URI.parse url_virtwl
+    #filename_virtwl = 'virtwl.h_base64'
+    #sha256sum_virtwl = 'a8215f4946ccf30cbd61fcf2ecc4edfe6d05bffeee0bacadd910455274955446'
+    
+    #puts "Downloading virtwl".yellow
+    #system('curl', '-s', '-C', '-', '--insecure', '-L', '-#', url_virtwl, '-o', filename_virtwl)
+    #abort 'Checksum mismatch. :/ Try again.'.lightred unless
+      #Digest::SHA256.hexdigest( File.read( filename_virtwl ) ) == sha256sum_virtwl
+    #puts "virtwl base64 downloaded".lightgreen
+    #FileUtils.mkdir_p 'build/linux'
+    #system 'base64 --decode virtwl.h_base64 > build/linux/virtwl.h'   
+    
```

Comment:
No sha265sum check?

Reply:
The question is how?

The Google source servers seem to tar on the fly, which makes  for nonreproducible archives which don't have consistent sha256 sums.

Reply:
In case it isn't obvious, sommelier as of this commit is moving BACK to including virtwl.h, so no additional checksumming of that file is needed.